### DISCUSSION TITLE
Support MulticlassF1Score in AutoMM

### DISCUSSION THREAD

**Discussion by rsj123** [autogluon/autogluon#3413] [Ideas]
I am trying to train some model with nlp prolem.
Accuracy is not satisfy my requirements. and f1 macro is not supported by torchmetrics.
So is it possible to support MulticlassF1Score. So we can use it in multiclass train?

**Comment by zhiqiangdon** [comment]
Hi, try our 1.0 release, which should be able to support this. Thanks.
